Dokument/Dateien/Magie_Essenz.tex

```diff
@@ -1,103 +1,870 @@
+	\1 ...
+\end{outline}
+
+\subsubsection{Meisterliche Beherrschung} 
+\begin{outline}
+	\1 Wahre Meister des Fachs können das Licht so geschickt manipulieren, dass sie einem unaufmerksamen Auge gar als unsichtbar scheinen können.
+	\1 Die Kontrolle über die An- oder Abwesenheit von Licht in einem gewissen Bereich.
+	\1 ...
+\end{outline}
+Zurück zu Tab. \ref{tab:magie-verbreitung}.
+
+
+
+\subsection{Mikrowellen}\label{magie:mikrowelle}
+Mit Mikrowellen lässt sich Wasser erhitzen und abkühlen, denn diese treffen genau die Eigenfrequenz des Wassers.
+Das gilt demnach für alles, was Wassermoleküle enthält.
+Weiterhin ermöglichen Mikrowellen ein Radarsystem.
+
+\subsubsection{Verbreitung}
+Vor allem Tiere.
+
+\subsubsection{Bedeutung im Spiel}
+...
+
+\subsubsection{Intuitive Nutzung}
+Viele Tiere mit dieser Eigenschaft haben parallel ein Radarsystem entwickelt.
+Die Sensoren befinden sich mit bei den Ohren.
+Das erlaubt Herdentieren oder Rudel, sich auch über weitere Strecken hinweg zu lokalisieren, wiederzufinden oder in Bezug auf Beute zu wissen, in welche Richtung ihr noch niemand auflauert.
+
+\subsubsection{Erlernbare Steigerungen}
+\begin{outline}
+	\1 ...
+\end{outline}
+
+\subsubsection{Meisterliche Beherrschung} 
+\begin{outline}
+	\1 ...
+\end{outline}
+Zurück zu Tab. \ref{tab:magie-verbreitung}.
+
 
-\subsubsection{reine Energie}\label{sec:energiemagie}
+
+\subsection{Proliferation}\label{magie:proliferation}
+Diese Eigenschaft ist eher chemischer Natur und erlaubt die Beeinflussung der Reaktionsgeschwindigkeit.
+Aus diesem Grunde leitet sie sich auch von der \nameref{magie:temperatur} und dem \nameref{magie:gleichstrom} ab.
+Ersteres hat direkten Einfluss auf die Geschwindigkeit: bei erhöhter Temperatur laufen Reaktionen schneller ab, da sich Teilchen schneller bewegen.
+Der Gleichstrom dient als Katalysator und senkt die Energiebarriere die dem Start der Reaktion entgegensteht.
+Folgen von beschleunigten Reaktionen findet sich dann auch bei den Zellen wieder: schnelleres Wachstum führt zum Beispiel zu einem schnelleren Lebenszyklus oder zur beschleunigten Reparatur von Schäden - auch im gesamten Gewebe.
+
+\subsubsection{Verbreitung}
+Wenige Pflanzen- und Tierarten sind dieser Fähigkeit mächtig. 
+Auch bei den Hominini ist es eine seltene Gabe. 
+Denn diese Fähigkeit leitet sich zwar von den recht häufig verbreiteten Arten Temperatur und Gleichstrom ab, doch nicht immer, wenn im Nachfahren diese Kombination vorliegt, beherrscht dieser die Proliferation.
+Daher ist sie so selten und hoch geschätzt.
+Diese Hominini leben länger als ihre Verwandten, abhängig von der Intensität ihrer Begabung sogar bis zu \SI{200}{\percent} der durchschnittlichen Lebensdauer.
+Allerdings kommt mit dieser Fähigkeit ein ungünstiger Nachteil: sie sind unfähig, sich mit ihren Artangehörigen zu vermehren, da ihre Keimzellen "`nicht auf gleicher Geschwindigkeit laufen"'.
```

Comment:
ich würde das etwas abschwächen, sodass es schwieriger ist, sich zu vermehren, je stärker die Proliferations-Fähigkeit in einem manifestiert ist.

Reply:
Warum? Dann ist das züchten ja immer noch möglich...
Also wie lauten deine Gründe? :D

Reply:
naja, meiner vorstellung nach sollten halt langsamere zellprozesse nicht automatisch zu unfruchtbarkeit führen. das ist ja nicht wie ein on/off-schalter.

Reply:
siehe Discord